lib/cisco_node_utils/interface.rb

```diff
@@ -283,7 +289,7 @@ def default_duplex
 
     def negotiate_auto_lookup_string
       case @name
-      when /Ethernet/i
+      when /Ethernet/i, /GigE/i, /MgmtEth/i
```

Comment:
That's why I used /GigE/ with no anchors - it matches `TenGigE` and `HundredGigE`. `GigabitEthernet` is matched by the existing /Ethernet/ regexp.